app/src/main/assets/ime/keyboard/org.florisboard.layouts/layouts/characters/bepo_new.json

```
@@ -0,0 +1,43 @@
+[
+  [
+    { "$": "auto_text_key", "code":   98, "label": "b" },
+    { "$": "auto_text_key", "code":  233, "label": "é" },
+    { "$": "auto_text_key", "code":  112, "label": "p" },
+    { "$": "auto_text_key", "code":  111, "label": "o" },
+    { "$": "auto_text_key", "code":  118, "label": "v" },
+    { "$": "auto_text_key", "code":  100, "label": "d" },
+    { "$": "auto_text_key", "code":  108, "label": "l" },
+    { "$": "auto_text_key", "code":  106, "label": "j" },
+    { "$": "auto_text_key", "code":  122, "label": "z" },
+    { "$": "auto_text_key", "code":  119, "label": "w" }
+  ],
+  [
+    { "$": "auto_text_key", "code":   97, "label": "a" },
+    { "$": "auto_text_key", "code":  117, "label": "u" },
+    { "$": "auto_text_key", "code":  105, "label": "i" },
+    { "$": "auto_text_key", "code":  101, "label": "e" },
+    { "$": "auto_text_key", "code":   99, "label": "c" },
+    { "$": "auto_text_key", "code":  116, "label": "t" },
+    { "$": "auto_text_key", "code":  115, "label": "s" },
+    { "$": "auto_text_key", "code":  114, "label": "r" },
+    { "$": "auto_text_key", "code":  110, "label": "n" },
+    { "$": "auto_text_key", "code":  109, "label": "m" },
```

```suggestion
    { "$": "auto_text_key", "code":  109, "label": "m" }
```

There is an additional comma which shouldn't be here